libdnf/dnf-keyring.cpp

```diff
@@ -271,7 +270,7 @@ dnf_keyring_check_untrusted_file(rpmKeyring keyring,
         g_set_error_literal(error, DNF_ERROR, DNF_ERROR_INTERNAL_ERROR, "failed to set keyring");
         goto out;
     }
-    rpmtsSetVfyLevel(ts, RPMSIG_SIGNATURE_TYPE);
+    rpmtsSetVfyLevel(ts, RPMSIG_SIGNATURE_TYPE|RPMSIG_DIGEST_TYPE);
```

Comment:
I was reading the related [PR](https://github.com/rpm-software-management/libdnf/pull/1179) where this change was originally proposed. So can we assume that there are already none packages with MD5 digest present on the supported systems or we can consider them untrusted?